src/main/java/com/github/princesslana/eriscasper/rest/emoji/CreateEmojiRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.github.princesslana.eriscasper.rest.emoji;
+
+import com.github.princesslana.eriscasper.data.Snowflake;
+import com.google.common.collect.ImmutableList;
+import org.immutables.value.Value;
+
+/**
+ * @see <a
+ *     href="https://discordapp.com/developers/docs/resources/emoji#create-guild-emoji-json-params">
+ *     https://discordapp.com/developers/docs/resources/emoji#create-guild-emoji-json-params</a>
+ */
+@Value.Style(deepImmutablesDetection = true)
+@Value.Immutable
+public interface CreateEmojiRequest {
+  String getName();
+
+  EmojiImageWrapper getImage();
```

Comment:
I'm undecided on this. I think my preferred way is to have this be String, and provide convenience methods that allow a File or whatever to be passed in.

Reply:
you can use a string in the current setup